app-config.yaml

```diff
@@ -18,7 +18,7 @@ proxy:
     target: https://circleci.com/api/v1.1
     changeOrigin: true
     pathRewrite:
-      '^/proxy/circleci/api/': '/'
+      '^/api/proxy/circleci/api/': '/'
```

Comment:
is this even needed? 

isn't it the default? the Jenkins plugin works without it

Reply:
Apparently 🤷. I just confirmed again on master it only works by adding the `/api`.

Reply:
i meant removing path rewrite, is that what you tried?

Reply:
Yeah, your completely right. If I remove the `pathRewrite` and `changeOrigin` works fine. Thanks.